tests/exec.go

```diff
@@ -38,7 +38,7 @@ func Exec(o *option.Option) {
 				gomega.Expect(output).Should(gomega.Equal(strEchoed))
 			})
 
-			for _, interactive := range []string{"-i", "--interactive"} {
+			for _, interactive := range []string{"-i", "--interactive", "-i=true", "--interactive=true"} {
```

Comment:
I wonder if it's worth compiling a list of all possible truthy operations (e.g. true, TRUE, 1, etc.) and running it against all bool flags. Maybe too big of a scope for this PR, though, so I'm fine if we defer it to another PR

Reply:
we do track some of these truthy values in the unit tests.

Reply:
Yeah, I just wonder if we could have a more comprehensive list so we're not appending them manually for every flag (current and future) that allows for booleans.

Anyway, again not really an issue for PR, just something I think we should consider later on.